mysql-test/suite/innodb/r/log_encrypt_kill.result

```diff
@@ -6,6 +6,8 @@ SET GLOBAL innodb_redo_log_encrypt = 1;
 Warnings:
 Warning	13068	InnoDB: Can't set redo log tablespace to be encrypted.
 SET GLOBAL innodb_undo_log_encrypt = 1;
```

Comment:
Can you do SELECT on the global variable after this operation?

two things
1. it should be error message (if possible)
2. the value should be OFF/0 after the SET failed.

Reply:
1 - all other similar messages are warnings (because that's how existing upstream messages worked around this)
2 - fixed

Reply:
Right, I think we agreed that we will deviate this from upstream? And warning to error deviation makes sense IMHO.

Reply:
I mean, all other messages in our codebase (for redo encryption) are warnings currently. Upstream reports a warning if the server is read only, and doesn't notify the user for any other failure. We changed the code so we report warnings for other cases too.

Reply:
If it is simple change, I would like you to try it in this PR

Reply:
As we discussed on slack: I can make most of the messages errors, but not all of them - we can't report errors from the update function, only from validate. I could move most of the checks to validate, but it'll be more complex there, and I still would have to keep some warnings. So it's best to leave these as is.